strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceImpl.java

```diff
@@ -57,6 +57,8 @@
     public <S extends ArtifactEntry> S save(S entity,
                                             boolean updateLastVersion)
     {
+        entity.setArtifactCoordinates(entity.getArtifactCoordinates());
```

Comment:
self replacement

Reply:
yeah, it has a point to be

Reply:
Could you please explain `why` we need such things ? Will we have to do similar things for other entities ?

Reply:
I leave the comment there, this is needed to reset `path` value of `ArtifactEntry`.